actionpack/lib/action_dispatch/http/mime_types.rb

```diff
@@ -21,7 +21,7 @@
 Mime::Type.register "application/xml", :xml, %w( text/xml application/x-xml )
```

Comment:
If it is nevermind, just not clear from this line `Mime::Type.register "application/xml", :xml, %w( text/xml application/x-xml )` as it's not passed as the extension argument.